src/frequenz/sdk/actor/_runner.py

```diff
@@ -0,0 +1,27 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Utility functions to run and synchronize the execution of actors."""
```

Comment:
Hmm maybe a name like `_run_utils` or `async_utils` would be more descriptive?

Reply:
(for the file)

Reply:
Then I'll rename it to `_run_utils` as that's the name you kind of both suggested

Reply:
done